Makefile

```diff
@@ -1,10 +1,15 @@
-.PHONY: test reset
+.PHONY: test test_simple reset
 
 test:
 	docker-compose build
 	docker-compose run re_api sh scripts/run_tests.sh
 	docker-compose down --remove-orphans
 
+test_simple:
```

Comment:
Should we leave this in? I don't know how everyone else does it! How do they live without pytest

Reply:
I don't think it's all that useful as someone could pretty easily just edit the existing `run_tests.sh` script to single out specific tests to run

Reply:
Yeah no ... this is a PITA

Reply:
I like the pytest output. It has colors and shows the parameters passed in along the stack